azure-pipelines-pr-validation.yml

```diff
@@ -194,3 +194,30 @@ stages:
     - task: ms-vseng.MicroBuildTasks.521a94ea-9e68-468a-8167-6dcf361ea776.MicroBuildCleanup@1
       displayName: Perform Cleanup Tasks
       condition: succeededOrFailed()
+
+- stage: insert
+  displayName: Create Insertion
+  dependsOn:
+    - build
+  
+  jobs:
+  - job: insert
+    displayName: Insert to VS
+    pool:
+      vmImage: windows-2019
+    steps:
+    - powershell: Write-Host "##vso[task.setvariable variable=SourceBranchName]$('$(Build.SourceBranch)'.Substring('refs/heads/'.Length))"
+      displayName: Setting SourceBranchName variable
+      condition: succeeded()
+      
+    - template: eng/pipelines/insert.yml
+      parameters:
+        createDraftPR: true
+        autoComplete: false
+        insertToolset: $(InsertToolset)
+        clientId: $(ClientId)
+        clientSecret: $(ClientSecret)
+        vsBranchName: $(VisualStudioBranchName)
+        titlePrefix: $(OptionalTitlePrefix)
+        sourceBranch: $(SourceBranchName)
```

Comment:
Should we surface any of these as pipeline parameters?

Reply:
They are, just added via the UI because you can't set defaults in yaml and then override them
That is where the VisualStudioBranchName parameter is coming from, e.g.
![image](https://user-images.githubusercontent.com/5749229/127244890-f73116c0-2897-46b9-8da9-a845a96d3dd5.png)

Reply:
Not even since we are passing them unmodified as parameters to the insert template?